step.yml

```diff
@@ -0,0 +1,85 @@
+title: |-
+  Simple Git Clone
+summary: |-
+  The step checks out the defined repository state without interfering with environmental variables.
+description: |
+  The checkout process depends on the checkout properties: the Step checks out a repository state defined by a branch, git commit or a git tag.
+
+  ### Configuring the Step
+
+  1. The **Git repository URL** is required field, the address of the repository that needs to be cloned.
+  2. The **Clone destination** is required field, the local path where we need to clone. This is set by default to *_tmp* in current work directory
+  3. In the **Clone Config** section there should be exactly one set of the following:
+     - **Branch**
+     - **Tag**
+     - **Commit**
+
+  ### Related Steps
+
+  - [Activate SSH key (RSA private key)](https://www.bitrise.io/integrations/steps/activate-ssh-key)
+  - [Bitrise.io Cache:Pull](https://www.bitrise.io/integrations/steps/cache-pull)
+  - [Bitrise.io Cache:Push](https://www.bitrise.io/integrations/steps/cache-push)
+  - [Git Clone Repository](https://www.bitrise.io/integrations/steps/git-clone)
+website: https://github.com/bitrise-steplib/bitrise-step-simple-git-clone
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-simple-git-clone
+support_url: https://github.com/bitrise-steplib/bitrise-step-simple-git-clone/issues
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+type_tags:
+  - utility
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+deps:
+  brew:
+    - name: git
+    - name: wget
+  apt_get:
+    - name: git
+    - name: wget
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-simple-git-clone
+inputs:
+  - repository_url:
+    opts:
+      title: "Git repository URL"
+      summary: |-
+        URL of the git repository you wish to clone.
+      description: |-
+        The step will try to fetch the repository from the given URL and clone it to the defined clone directory.
+      is_required: true
+  - clone_into_dir: "$BITRISE_SOURCE_DIR/_tmp"
+    opts:
+      title: "Clone destination (local) directory path"
+      summary: |-
+        Path of the directory where the cloned repository will be placed.
+      description: |-
+        Path of the directory where the cloned repository will be placed.
+      is_required: true
+  - commit:
+    opts:
+      category: Clone Config
```

Comment:
no need to hide these inputs with a category: https://devcenter.bitrise.io/contributors/create-your-own-step/#grouping-inputs-together